test/parallel/test-event-emitter-add-listeners.js

```diff
@@ -68,8 +68,8 @@ const EventEmitter = require('events');
 }
 
 {
-  const listen1 = function listen1() {};
-  const listen2 = function listen2() {};
+  const listen1 = () => {};
```

Comment:
Definitely not here or on the line below, and this is actually something that I've worried about for a while: `common.noop` doesn't really save us a lot of code and can lead to subtle bugs such as if it is used here.

If used here and the line below, then `listen1` and `listen2` are the same object, thus invalidating the test!

This could perhaps be fixed by making `common.noop` a getter that returns a different no-op function object each time it is called. But I kinda wonder if we might just be better off doing a global find/replace on `common.noop` and using `() => {}` instead.

/cc @nodejs/testing